NuRadioMC/test/examples/test_webinar.sh

```diff
@@ -9,5 +9,5 @@ python3 W04EffectiveVolumes.py
 python3 W05ElectricFields.py
 rm -r results
 rm input_3.2e+17_1.0e+19.hdf5
-rm input_3.2e+18_1.0e+19.hdf5.part000?
+rm input_3.2e+18_1.0e+19.hdf5.part00??
```

Comment:
I'm guessing you noticed that this produced more than 10 files, so the clean-up didn't cover all of them. I've noticed the same thing and don't really understand it; I feel like there shouldn't be _this_ many events. It might be good if someone could look into this, but this is probably outside the scope of this PR.